src/cli/task-telemetry.ts

```diff
@@ -0,0 +1,36 @@
+import type { Config} from '../declarations';
+import { checkTelemetry, disableTelemetry, enableTelemetry } from './telemetry/telemetry';
+
+export const taskTelemetry = async (config: Config) => {
+  const p = config.logger.dim(config.sys.details.platform === 'windows' ? '>' : '$');
+  const logger = config.logger;
+  const isEnabling = config.flags.args.includes("on");
+  const isDisabling = config.flags.args.includes("off");
+  const INFORMATION = `Opt in or our of telemetry. Information about the data we collect is available on our website: ${logger.bold('https://stenciljs.com/telemetry')}`
+  const THANK_YOU = `Thank you for helping to make Stencil better! 💖`;
+  const ENABLED_MESSAGE = `${logger.green('Enabled')}. ${THANK_YOU}\n\n`;
+  const DISABLED_MESSAGE = `${logger.red('Disabled')}\n`;
+  let hasTelemetry = await checkTelemetry();
+
+  if (isEnabling) {
+    await enableTelemetry()
+    console.log(`\n  ${logger.bold('Telemetry is now ') + ENABLED_MESSAGE}`)
+    return;
+  }
+
+  if (isDisabling) {
+    await disableTelemetry()
```

Comment:
What happens if this fails? Right now, we only `console.error` the `Error` that occurred. As a result, we could fail to write the file, but report to the user we set the value correctly (erroneously)